x-pack/test/security_solution_cypress/cypress/e2e/cloud_security_posture/host_misconfiguration_preview.cy.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
should we add our team as code owners of 
`x-pack/test/security_solution_cypress/cypress/e2e/cloud_security_posture`?

Reply:
As per discussion we will do the cypress test and anything related to it to different PR